docs/Secure-Coding-Guide-for-Python/CWE-707/CWE-175/README.md

```diff
@@ -0,0 +1,233 @@
+# CWE-175: Improper Handling of Mixed Encoding
+
+Locale-dependent programs may produce unexpected behavior or security bypasses in an environment whose locale is unset, or not set to an appropriate value.
+
+In Python, the `locale` module will never perform case conversions or character classifications according to the locale [[Python 3.9 locale](https://docs.python.org/3.9/library/locale.html)]. Python follows the Unicode conventions for text [[Python Issue 34928](https://bugs.python.org/issue34928)], and this behavior may not match the user's expectation in their native language. Some characters, such as the German ß (eszett), Greek Σ (sigma) also have particular rules in Unicode that may not match native usage and need to be handled carefully. Also, be aware of differences in other languages that use right-to-left ordering (e.g. Arabic) and languages which use CJK characters.
+
+>[!NOTE]
+> Running the code examples for this rule might generate the following error message: `locale.Error: unsupported locale setting`
+> It's usually thrown when the locales used for the examples are not available on the OS. For these examples, you will need the following locales:
+>
+> * de_DE.utf8
+> * en_IE.utf8
+> * tr_TR.utf8
+> * uk_UA.utf8
+>
+> The method of installing locales depends on the OS. To install locales on Debian, do the following:
+>
+> 1. run `sudo dpkg-reconfigure locales`
+> 2. select the needed locales (make sure they are uncommented in /etc/locale.gen)
+> 3. run `sudo locale-gen`
+
+For example, Python does not handle the Turkish dotted-i as expected.
+
+[*example01.py:*](example01.py)
+
+```py
+""" Code Example """
+import locale
+word = "Title"
```

Comment:
Getting:
Constant name "word" doesn't conform to UPPER_CASE naming stylePylint[C0103:invalid-name](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/invalid-name.html)
Maybe just capitalise "word" variable to silence the linters?